tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/journal/impl/TimedBufferTest.java

```diff
@@ -205,162 +203,4 @@ public int getRemainingBytes() {
       }
 
    }
-
-   /**
-    * This test will verify if the system will switch to spin case the system can't perform sleeps timely
-    * due to proper kernel installations
-    *
-    * @throws Exception
-    */
-   @Test
```

Comment:
Because the new implementation rely on LockSupport.parkNanos to wait (is a MONOTONIC clock on linux AFAIK) and it doesn't wait never the whole time until the given expiration.
In addition the background flusher use the Semaphore only after a "long" period of no writes.
The only practical reason that could lead it to not be triggered in time is dependent by the writers: if a writers wait a lot to push data, the time could expire, but I think is normal no?